mitodl/odl-video-service
# Simple video player

#### What are the relevant tickets?
https://github.com/mitodl/odl-video-service/issues/20

#### What's this PR do?
Adds a 'VideoPlayer' React component using video.js to play a video on the video detail page.

#### How should this be manually tested?
Upload a new video file.  Wait a few minutes for it to be transcoded. View the video's detail page.  The video should appear and automatically start playing.  There should be a button to manually choose the resolution.
